source/message/OP_MSG.rst

```diff
@@ -206,6 +215,40 @@ connection. All other errors during processing will be silently dropped, and
 will not result in the connection being closed.
 
 
+moreToCome On Responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When the ``moreToCome`` flag is set on a response it signals to the recipient
+that the sender will send additional responses on the connection. The recipient
+MUST continue to read responses until it reads a response with the ``moreToCome``
+flag not set, and MUST NOT send any more requests on this connection until
+it reads a response with the ``moreToCome`` flag not set. The client MUST
+either consume all messages with ``moreToCome`` flag set or close the connection.
+
+When the server sends responses with ``moreToCome`` flag set, it behaves as if
+the client sent a ``getMore`` request for the returned cursor for each of the
+responses. Each reply will have a unique ``messageId``, and the ``responseTo``
```

Comment:
> When the server sends responses with ``moreToCome`` flag set, it behaves as if the client sent a ``getMore`` request for the returned cursor for each of the responses.

Is this necessary? Perhaps we can say that server responses with the ``moreToCome`` flag will be structured similarly to normal ``getMore`` responses. I believe this relates to the client specifying ``exhaustAllowed`` on the preceding ``find`` command, but I understand if you'd rather not mention that here since you're going to discuss it in the next section.

Reply:
Removed this entirely.